python/cudf/cudf/io/parquet.py

```diff
@@ -480,6 +482,8 @@ def to_parquet(
                     statistics=statistics,
                     metadata_file_path=metadata_file_path,
                     int96_timestamps=int96_timestamps,
+                    row_group_size_bytes=row_group_size_bytes,
+                    row_group_size_rows=row_group_size_rows,
```

Comment:
Do we need to document these in the docstring?

Reply:
I always forget to update the docs! Thanks for the reminder.